tailwind.config.js

```diff
@@ -1,11 +1,26 @@
 const colors = require('tailwindcss/colors');
 
 module.exports = {
-  mode: 'jit',
```

Comment:
Waarom zou je JIT mode uitzetten? 😬

Reply:
Hij  update mijn styles  toen niet als ik iets wou proberen. Als ik deze modus had afgelegd werkte het wel  weer terug daarmee 😄

Reply:
Met custom styles die hij niet herkende? Want ik heb soms ook wel dat ik 2x moet saven voordat hij het herkent, is denk ik nog een bugje in Tailwind zelf.

Reply:
En wou je dat in browser inspector proberen?